modules/_template.nix

```diff
@@ -1,4 +1,14 @@
-{ self, lib, ... }: {
-  perSystem = { config, self', inputs', pkgs, ... }: { };
-  flake = { };
+{
+  self,
+  lib,
+  ...
+}: {
+  perSystem = {
+    config,
```

Comment:
can alejandra be configured to keep the commas at the beginning of the line?

Reply:
Being configurable would go against the purpose of a formatter, which is to have the same format everywhere in the ecosystem.
Alejandra does not use leading commas. More details about that decision can be found here: https://github.com/kamadorueda/alejandra/issues/95

Reply:
what happens if you add a trailing comma, does it remote it?

Reply:
Trailing commas are fine for alejandra. I have the habit to put them everywhere for consistency.

Reply:
yes i was wondering if it adds them when they're missing! i guess we'll find out